js/utility.js

```diff
@@ -36,6 +36,24 @@ const Utils = {
     }
 
     return ``;
+  },
+
+  getSlugFromHelpTag(helpTag) {
+    // turns help type into a prettier string so we can use it as slug
+    // e.g., "Plan & organize" becomes "plan-and-organize"
+
+    return helpTag.split(` `)
+      .map(word => encodeURIComponent(word.replace(/&/g, `and`)).toLowerCase())
+      .join(`-`);
+  },
+
+  getHelpTagFromSlug(slug) {
+    // turns help type slug into a string Pulse API recognizes as helpType
+    // e.g., "plan-and-organize" becomes "plan & organize"
+
+    return slug.split(`-`)
+      .map(word => decodeURIComponent(word.replace(/and/g, `&`)))
```

Comment:
this will turn "grand" into "gr&", you probably want `word.replace(/\band\b/g, `&`)` here which only replaces `and` if it has word boundaries on either side (such as start/end of phrase, whitespace, etc).

Reply:
yay regex master to rescue!